### ISSUE TITLE
make an Icon component

### ISSUE THREAD

**Issue by marwahaha** [tgeorgeux/new.jupyter.github.io#30]
It should specify a size and "icon name" and load it. 

We can put the "clickable" logic in here, too (i.e. for the footer), but I'm not sure it's necessary

Also, replace the Footer's custom images with Icon components.
depends on #27

**Comment by tgeorgeux** [comment]
For now let's use size 36x36 for all icons.

**Comment by marwahaha** [comment]
Won't do
